docker/Dockerfile.intel

```diff
@@ -0,0 +1,90 @@
+##############################################################
```

Comment:
there's a massive amount of code duplication here -- sigh

as I pointed out in Italo's PR for OpenMP offloading, which should rather establish a hierarchy of dockerfiles and reuse code as much as possible

we don't duplicate code -- ever !